docs/sphinx/dev_guide/ci_tasks.rst

```diff
@@ -157,6 +157,113 @@ annotate the job for this. For example:
    describe the change in the ``RAJA/.gitlab/jobs/<MACHINE>.yml`` file where
    the job is overridden.
 
+
+Building the Intel clang + SYCL HIP compiler for use in CI
+------------------------------------------------------
+
+The SYCL CI tests on corona rely on a custom Intel Clang SYCL compiler that we 
+build ourselves. This compiler lives in the /usr/workspace/raja-dev/ folder so 
+that it can be accessed by the gitlab CI system. Since the intel compiler does
+not do releases in the typical sense (they simply update their repo *every night*), 
+it may become necessary to periodically build a new version of the compiler to 
+ensure that we are using the most up-to-date version available. The steps for 
+building/installing/running are shown here.
+
+Building the Compiler
+^^^^^^^^^^^^^^^^^^^^^
+
+.. important:: Because intel update's their compiler repo daily, there is a nonzero possibility that the had of the sycl branch will build. 
+  In the event that it does not build, try checking out a different commit. On the intel/llvm github page, one can see which of their 
+  commits builds by checking the status badge next to each commit. Look for a commit that passes. 
+
+#. 1. Load the version of GCC that you want to use. in this case, we are using LC's gcc/10.3.1-magic installation. 
+
+  .. code-block:: bash
+
+    module load gcc/10.3.1-magic
+
+#. 2. Clone the "sycl" branch of intel's llvm compiler fork 
+
+  .. code-block:: bash
+
+    git clone https://github.com/intel/llvm -b sycl
+
+#. 3. cd into that folder 
+
+  .. code-block:: bash
+    
+    cd llvm
+
+  #. 3a. In the event that the head of the sycl branch does not build, run `git checkout <git sha>` to checkout a version that does build.
+
+#. 4. build the compiler. Note that in this example, we are using rocm5.7.1, but one can change the version they wish to use simply by changing the paths in the configure step.
```

Comment:
Does  the rocm version module need to be loaded? I think it does, but I may be wrong.

Reply:
Oh yeah, you're right, it does